zipline/finance/blotter/__init__.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright 2018 Quantopian, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from .simulatedblotter import SimulatedBlotter
+from .core import (
+    blotters,
+    register,
+    unregister,
+    load,
+)
+
+__all__ = ['SimulatedBlotter', 'unregister', 'blotters', 'load']
+
+
+@register('default')
```

Comment:
should we just decorate the definition of the class itself?

Reply:
Yes, that could be another option - but again, similarly to the first comment at the very top, if we switch to handling classes getting passed in from decorators, that's fine, but I'm not sure we would be able to support user-defined functions getting passed in at the same time... unless we don't want to do that or if we distinguish by type of thing being passed in, that is - does that sound like something we would want to have?